app/src/main/java/fr/free/nrw/commons/review/ReviewController.java

```diff
@@ -38,6 +42,7 @@
     private NotificationManager notificationManager;
     private NotificationCompat.Builder notificationBuilder;
     private Media media;
+    private static final String THANKS_LOG="https://en.wikipedia.org/wiki/Special:Log/thanks";
 
```

Comment:
It should go to https://commons.wikimedia.org/wiki/Special:Log/thanks or https://commons.wikimedia.beta.wmflabs.org/wiki/Special:Log/thanks instead, depending to the beta flavor.